incident/mysql/incident.go

```diff
@@ -35,7 +35,7 @@ type mysqlReporter struct {
 // in a MySQL database, all of which will be marked as emanating from the given
 // source.
 func NewMySQLReporter(ctx context.Context, db *sql.DB, source string) (incident.Reporter, error) {
-	stmt, err := db.PrepareContext(ctx, "INSERT INTO Incidents(Timestamp, Source, BaseURL, Summary, IsViolation, FullURL, Details) VALUES (?, ?, ?, ?, ?, ?, ?, ?);")
+	stmt, err := db.PrepareContext(ctx, "INSERT INTO Incidents(Timestamp, Source, BaseURL, Summary, IsViolation, FullURL, Details) VALUES (?, ?, ?, ?, ?, ?, ?);")
```

Comment:
I'm surprised the old code worked, with the number of parameters not matching the number of columns.

Reply:
It just didn't. As the tests on CI detect no mysql instance available and don't run any of db tests.

Reply:
Ah we should fix that! It's pretty easy to get Travis to install a MySQL server.

Reply:
Let's chat on that. I'll make a separate PR.

Reply:
https://github.com/google/monologue/pull/67